src/test/java/cloud/eppo/BaseEppoClientTest.java

```diff
@@ -66,7 +66,7 @@ private void initClientWithData(
         new BaseEppoClient(
             DUMMY_FLAG_API_KEY,
             isConfigObfuscated ? "android" : "java",
-            "3.0.0",
+            "100.1.0",
```

Comment:
This version string does not need to equal what version the package is actually at. setting the test string to `100.1.0` makes it more clear than `3.0.0` which is close to actual.